homeassistant/components/denonavr/media_player.py

```diff
@@ -133,6 +153,31 @@ async def async_setup_entry(
         f"async_{SERVICE_UPDATE_AUDYSSEY}",
     )
 
+    async def async_service_handle(service_call: ServiceCall):
+        """Handle dispatched services."""
+        assert platform is not None
+        entities = await platform.async_extract_from_service(service_call)
+
+        response = {}
+
+        for entity in entities:
+            assert isinstance(entity, DenonDevice)
+
+            if service_call.service == SERVICE_TELNET_COMMANDS:
+                response[entity.entity_id] = await entity.async_send_telnet_commands(
```

Comment:
I'm a bit confused 😕 
I started by using `register_entity_service` because it is very easy to implement. However, it does not support returning any values, so I switched generic service register function, which seems to be the way to go [according to the developer docs](https://developers.home-assistant.io/docs/dev_101_services#response-data).
So, here is my question. How should I implement a service for an entity which returns data?